.gitignore

```diff
@@ -1,3 +1,6 @@
+bank stuff
+cldf/.transcription-report.json
```

Comment:
we do not ignore the transcription report usually, but upload it, right @xrotwang ?

Reply:
This line was auto-generated, I will change it, but maybe it is possible to change `cldfbench new` so that this is not ignored by default anymore ?